app/controllers/ht_approval_requests_controller.rb

```diff
@@ -18,8 +18,9 @@ def create # rubocop:disable Metrics/MethodLength
   end
 
   def index
-    reqs = HTApprovalRequest.order('approver')
-    @reqs = reqs.map { |r| HTApprovalRequestPresenter.new(r) }
+    requests = HTApprovalRequest.order('approver')
+    @active_reqs = requests.not_renewed.map { |r| HTApprovalRequestPresenter.new(r) }
+    @inactive_reqs = requests.renewed.map { |r| HTApprovalRequestPresenter.new(r) }
```

Comment:
I wonder if "completed" might be a better term?

Reply:
I'll meet you half way with "complete" and "incomplete"

Reply:
:+1: